ui/src/LandingPage.js

```diff
@@ -335,6 +374,72 @@ class LandingPage extends Component {
     this.setState({ createPrintTemplateDialogDisplayed: false });
   };
 
+  onCreateSmsTemplate = async () => {
+    var failedValidation = false;
+
+    if (!this.state.packCode.trim()) {
+      this.setState({ packCodeValidationError: true });
+      failedValidation = true;
+    }
+    if (!this.state.notifyTemplateId) {
```

Comment:
would be nice if we validated that the template ID was a UUID

Reply:
Now checks if uuid